recipes/exspy/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "exspy" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/exspy-{{ version }}.tar.gz
+  sha256: 1d588e28397cd39bd7807b7c230598cede5d284c1d0ef2d16de06413df001b1a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=64
+    - setuptools-scm >=8
+    - pip
+  run:
+    - python >=3.8
+    - dask-core
+    - hyperspy-base >=2.0
+    - matplotlib-base
+    - numexpr
+    - numpy
+    - pint
+    - pooch
+    - prettytable
+    - requests
+    - scipy
+    - traits
+
+test:
+  imports:
+    - exspy
+  commands:
+    - pip check
+    - pytest --pyargs exspy
+  requires:
+    - pip
+    - pytest
+
+about:
+  summary: EELS and EDS analysis with the HyperSpy framework
+  home: https://github.com/hyperspy/exspy
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+  doc_url: https://hyperspy.org/exspy
+  dev_url: https://github.com/hyperspy/exspy
+
+extra:
+  recipe-maintainers:
+    - CSSFrancis
+    - ericpre
+    - francisco-dlp
+    - jlaehne
```

Comment:
@CSSFrancis, @francisco-dlp, @jlaehne, can you confirm that you are willing to be maintainers?

Reply:
Yes, I am happy to be a maintainer.